src/test/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceServiceSpec.scala

```diff
@@ -664,6 +664,40 @@ class ResourceServiceSpec extends FlatSpec with Matchers with ScalaFutures with
     res.futureValue(Timeout(Span(10, Seconds))) shouldBe expected
   }
 
+  it should "list required authDomains and missing authDomains if user is a member of a universal policy" in {
```

Comment:
what is a universal policy?

Reply:
I meant it as something like `admin-notifier` which will all users are a member of. any better name suggestions?

Reply:
just say `admin-notifier`